web/viewer-geckoview.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```

Comment:
The "regular" viewer has a BOM at the start of its HTML files, do we perhaps want to maintain that here as well?